src/styles/card.css

```diff
@@ -0,0 +1,93 @@
+
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  margin: auto;
+  background: var(--color-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 7px;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color-black);
+  margin-top: 40px;
+  width: 100%;
```

Comment:
it's redundant

Reply:
when styles were deleted, test has not passed

Reply:
No way! `h2` is a block element, so it has a width: 100% by default. Try it, please